.github/CODEOWNERS

```
@@ -22,6 +22,9 @@
 go.mod @grafana/backend-platform
 go.sum @grafana/backend-platform
 
+# Backend code docs
+/contribute/style-guides/backend.md
```

This might just be my limited knowledge of the CODEOWNERS syntax, but shouldn't this be

```suggestion
/contribute/style-guides/backend.md @grafana/backend-platform
```

---

Thanks @sakjur, I probably was too sleepy when adding this. It looks correct to me.